icepyx/core/convert.py

```diff
@@ -0,0 +1,209 @@
+import os
+import re
+import h5py
+import zarr
+import itertools
+import numpy as np
+
+# PURPOSE: Wrapper function for converting HDF5 files to different formats
+def file_converter(hdf5_file, path, reformat):
```

Comment:
What is the benefit of having these as a set of standalone functions rather than turning them into a conversion class?